app/views/incurred_incidentals/show.html.erb

```diff
@@ -68,16 +68,16 @@
         <div class="panel-heading">
           Documents
         </div>
-        <div class="panel-body no-gutter" style="max-height: 10;overflow-y: scroll;">
+        <div class="panel-body no-gutter documents" style="max-height: 10;overflow-y: scroll;">
```

Comment:
Let's seize this opportunity to extract this inline styling :) also, maybe look into best practices re: adding classes not used in CSS.